Expo1900/Expo1900/Controller/NavigationViewController.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  NavigationViewController.swift
+//  Expo1900
+//
+//  Created by Doogie, Red on 2022/04/21.
+//
+
+import UIKit
+
+class ExpoNavigationController: UINavigationController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        guard let _ = self.topViewController as? MainViewController else {
+            return .all
+        }
```

Comment:
타입 캐스팅한 값을 사용하지 않고 단순 타입 체크 정도만 하고 있으니 `as`보단 `is`가 더 적합할 것 같아요!

```suggestion
        guard topViewController is MainViewController else {
            return .all
        }
```